sonic-net/sonic-buildimage
# [202205] Update Broadcom DNX SAI version to 7.1.60.4-1

#### Why I did it
 Update Broadcom DNX SAI version to 7.1.60.4-1
Fixes: [16564](https://github.com/sonic-net/sonic-buildimage/issues/16564)

##### Work item tracking
- Microsoft ADO **(number only)**: 25245962

#### How I did it
Included fix in Broadcom SDK shared by Broadcom via this CSP : CS00012315073

#### How to verify it

On a multi-asic linecard following was verified during a 12hr observation.

On a LC with SAI having fix, syncd memory utilization don't increase over a period
```
root@svcstr-xxxx-lc1-1:/# which dpkg
/usr/bin/dpkg
root@svcstr-xxxx-lc1-1:/# /usr/bin/dpkg -l | grep saibcm
ii  libsaibcm                  7.1.60.4-1                     amd64        Switch Abstraction Interface sdk based on Broadcom SAI

docker stats

CONTAINER ID   NAME        CPU %     MEM USAGE / LIMIT     MEM %     NET I/O          BLOCK I/O         PIDS
fca5f2fb0367   syncd1      66.28%    1.133GiB / 31.25GiB   3.63%     746MB / 890MB    0B / 106kB        49
8cf3c003eb59   syncd0      74.85%    1.133GiB / 31.25GiB   3.63%     896MB / 1.56GB   176kB / 115kB     49

```

In another LC with SAI with no fix, we see syncd memory utilization increases
```

admin@svcstr-xxxx-lc2-1:~$ docker exec -it syncd0 bash
root@svcstr-xxxx-lc2-1:/# /usr/bin/dpkg -l | grep saibcm
ii  libsaibcm                  7.1.60.4                       amd64        Switch Abstraction Interface sdk based on Broadcom SAI

CONTAINER ID   NAME        CPU %     MEM USAGE / LIMIT     MEM %     NET I/O          BLOCK I/O         PIDS
8427adc8bfca   syncd1      71.58%    2.149GiB / 31.25GiB   6.88%     696MB / 822MB    175MB / 135kB     49
8b455ead2340   syncd0      69.48%    2.439GiB / 31.25GiB   7.81%     683MB / 1.29GB   97.1MB / 135kB    49
```

#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
